### PR TITLE
feat: @xstate/inspect support umd

### PR DESCRIPTION
**Pull request by lecepin**

### CODE REVIEW

File: packages/xstate-inspect/rollup.config.js [hunk 1]
```diff
@@ -30,6 +34,34 @@ const createNpmConfig = ({ input, output }) => ({
...
+    input,
+    external: makeExternalPredicate(Object.keys(pkg.peerDependencies)),
+    output: {
+      file,
+      format: 'umd',
+      name,
+      globals: {
+        react: 'React',
+        xstate: 'XState',
+        '@xstate/fsm': 'XStateFSM'
```
Comment (Andarist) [inline packages/xstate-inspect/rollup.config.js:48]: Does `@xstate/inspect` work with `@xstate`

Comment (Andarist) [inline packages/xstate-inspect/rollup.config.js:48]: I don't think that `@xstate/inspect` is using `@xstate/fsm` anyhow - not the latter is compatible with the inspect package
```suggestion
```

Comment (lecepin) [reply packages/xstate-inspect/rollup.config.js]: ok, fixed.

File: packages/xstate-inspect/package.json [hunk 2]
```diff
@@ -38,23 +38,28 @@
...
+    "rollup-plugin-terser": "^5.1.2",
     "rollup-plugin-typescript2": "^0.30.0",
     "typescript": "^4.5.2",
     "ws": "^8.4.0",
     "xstate": "*"
   },
   "peerDependencies": {
     "@types/ws": "^8.0.0",
-    "ws": "^8.0.0"
+    "fast-safe-stringify": "^2.0.7",
```
Comment (Andarist) [inline packages/xstate-inspect/package.json:55]: This should be moved back to dependencies as the `createUmdConfig` should "bundle" this in the final file.

**Pull request merged**